member/src/it/scala/MemberServerSpec.scala

```diff
@@ -0,0 +1,268 @@
+      val port = containers.container.getServicePort(serviceName, exposedPort)
+
+      val clientSettings: GrpcClientSettings = GrpcClientSettings.connectToServiceAt(host, port).withTls(false)
+      val client: MemberService = MemberServiceClient(clientSettings)
+
+      val response =
+        client.inactivateMember(InactivateMember(Some(memberId), Some(MemberId("ADMIN_3")))).futureValue
+      response.memberId should equal(Some(memberId))
+      validateMember(
+        Some(memberId),
+        Some(memberInfo),
+        Some(MemberId("ADMIN_1")),
+        Some(MemberId("ADMIN_3")),
+        MEMBER_STATUS_INACTIVE,
+        client
+      )
+    }
+  }
+
+  it should "handle grpc calls for Suspend Member" in {
+    withContainers {containers =>
+      val host = containers.container.getServiceHost(serviceName, exposedPort)
+      val port = containers.container.getServicePort(serviceName, exposedPort)
+
+      val clientSettings: GrpcClientSettings = GrpcClientSettings.connectToServiceAt(host, port).withTls(false)
+      val client: MemberService = MemberServiceClient(clientSettings)
+
+      val response =
+        client.suspendMember(SuspendMember(Some(memberId), Some(MemberId("ADMIN_4")))).futureValue
+      response.memberId should equal(Some(memberId))
+      validateMember(
+        Some(memberId),
+        Some(memberInfo),
+        Some(MemberId("ADMIN_1")),
+        Some(MemberId("ADMIN_4")),
+        MEMBER_STATUS_SUSPENDED,
+        client
+      )
+    }
+  }
+
+  it should "handle grpc calls for Update Member Info" in {
+    withContainers {containers =>
+      val host = containers.container.getServiceHost(serviceName, exposedPort)
+      val port = containers.container.getServicePort(serviceName, exposedPort)
+
+      val clientSettings: GrpcClientSettings = GrpcClientSettings.connectToServiceAt(host, port).withTls(false)
+      val client: MemberService = MemberServiceClient(clientSettings)
+
+      val response =
+        client
+          .updateMemberInfo(
+            UpdateMemberInfo(
+              Some(memberId),
+              Some(memberInfoWithMobNumber),
+              Some(MemberId("ADMIN_5"))
+            )
+          )
+          .futureValue
+      response.memberId should equal(Some(memberId))
+      validateMember(
+        Some(memberId),
+        Some(memberInfoWithMobNumber),
+        Some(MemberId("ADMIN_1")),
+        Some(MemberId("ADMIN_5")),
+        MEMBER_STATUS_SUSPENDED,
+        client
+      )
+    }
+  }
+
+  it should "handle grpc calls for Terminate Member" in {
+    withContainers {containers =>
+      val host = containers.container.getServiceHost(serviceName, exposedPort)
+      val port = containers.container.getServicePort(serviceName, exposedPort)
+
+      val clientSettings: GrpcClientSettings = GrpcClientSettings.connectToServiceAt(host, port).withTls(false)
+      val client: MemberService = MemberServiceClient(clientSettings)
+
+      val response =
```

Comment:
Let's keep this DRY. You've copy/pasted the above val statements 7 times.  Turn it into a function that returns a 7-tuple or otherwise simplify it.  Only write a piece of code once.

Reply:
Reduced code count by removing duplicated code